src/desktop/space/mod.rs

```diff
@@ -469,62 +343,29 @@ impl Space {
         );
 
         let layer_map = layer_map_for_output(output);
+        let mut space_elements: Vec<SpaceElements<'_, E>> = Vec::new();
 
-        let mut space_elements: Vec<SpaceElements<'_, C>> = Vec::new();
-
-        space_elements.extend(
-            custom_elements
-                .iter()
-                .map(SpaceElements::Custom)
-                .collect::<Vec<_>>(),
-        );
-
-        space_elements.extend(
-            self.windows()
-                .rev()
-                .map(SpaceElements::Window)
-                .collect::<Vec<_>>(),
-        );
+        space_elements.extend(self.elements.iter().rev().map(SpaceElements::Element));
```

Comment:
as we now use near to far for the render element we could possibly get rid of the `rev()` if we also store the element from near to far